Python/thread_pthread.h

```diff
@@ -12,6 +12,12 @@
 #endif
 #include <signal.h>
 
+#if defined(__linux__)
+#include <sys/syscall.h>
+#elif defined(__FreeBSD__)
+#include <pthread_np.h>
+#endif
```

Comment:
nitpick: I know that one of the few who like indenting macros for readbility, but I suggest:

```
#if defined(__linux__)
#  include <sys/syscall.h>   /* syscall(__NR_gettid) */
#elif defined(__FreeBSD__)
#  include <pthread_np.h>   /* pthread_getthreadid_np() */
#endif
```